### PR TITLE
Add telegraf 1.18.1 release notes

### PR DESCRIPTION
**Pull request by pierwill**

### CODE REVIEW

**Review by sjwang90 — APPROVED** [review]

File: content/telegraf/v1.18/about_the_project/release-notes-changelog.md [hunk 1]
```diff
@@ -7,6 +7,45 @@ menu:
...
+## v1.18.1 [2021-04-07]
+
+- Agent: Closes running outputs when agent reloads on SIGHUP.
+
+### Input plugin updates
+
+- [Docker Input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/docker)(`docker`):
+  Fixed panic when parsing container statistics.
+- [Exec Input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/exec)(`exec`):
+  Plugin does not truncate messages in debug mode.
```
Comment (kelseiv) [inline content/telegraf/v1.18/about_the_project/release-notes-changelog.md:20]: Truncate messages...
or
Prevent messages from being truncated...?

File: content/telegraf/v1.18/about_the_project/release-notes-changelog.md [hunk 2]
```diff
@@ -7,6 +7,45 @@ menu:
     weight: 10
     parent: About the project
 ---
+
+## v1.18.1 [2021-04-07]
+
+- Agent: Closes running outputs when agent reloads on SIGHUP.
+
+### Input plugin updates
+
```
Comment (kelseiv) [inline content/telegraf/v1.18/about_the_project/release-notes-changelog.md:16]: Fixed >> Fix

File: content/telegraf/v1.18/about_the_project/release-notes-changelog.md [hunk 3]
```diff
@@ -7,6 +7,45 @@ menu:
...
+- [Exec Input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/exec)(`exec`):
+  Plugin does not truncate messages in debug mode.
+- [IPMI Sensor Input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/ipmi_sensor)(`ipmi_sensor`):
+  Fixed panic by implementing a length check to plugin.
+- [MySQL Input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/mysql)(`mysql`):
+  Fixed the ability to handle ‘binary logs’ query for MySQL version 8.0+.
+- [NFS Client Input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/nfsclient)(`nfsclient`):
+  Fixed integer overflow in fields received by mountstat.
+- [Ping Input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/ping)(`ping`):
+  Fixed an issue when `method=”native”` by always calling `SetPrivileged(true)`.
```
Comment (kelseiv) [inline content/telegraf/v1.18/about_the_project/release-notes-changelog.md:28]: incomplete?

File: content/telegraf/v1.18/about_the_project/release-notes-changelog.md [hunk 4]
```diff
@@ -7,6 +7,45 @@ menu:
...
+- [SQL Server Input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/sqlserver)(`sqlserver`):
+  Fixed `sqlserver_process_cpu` calculation.
+- [Tail Input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/tail)(`tail`):
+  Added configurable option to override `path` tag.
+
+### Output plugin updates
+
+- [Azure Monitor Output](https://github.com/influxdata/telegraf/tree/master/plugins/outputs/azure_monitor)(`azure_monitor`):
+  Fixed an issue to handle error when initializing the authentication object.
+- [Yandex Cloud Output](https://github.com/influxdata/telegraf/tree/master/plugins/outputs/yandex_cloud_monitoring) (`yandex_cloud_monitoring`):
```
Comment (kelseiv) [inline content/telegraf/v1.18/about_the_project/release-notes-changelog.md:40]:
```suggestion
- [Yandex Cloud Monitoring Output](https://github.com/influxdata/telegraf/tree/master/plugins/outputs/yandex_cloud_monitoring) (`yandex_cloud_monitoring`):
```

Comment (kelseiv) [reply content/telegraf/v1.18/about_the_project/release-notes-changelog.md:40]: Let's add space after the rest of the plugins like we do here
Yandex Cloud Output (`yandex_cloud_monitoring`):

File: content/telegraf/v1.18/about_the_project/release-notes-changelog.md [hunk 5]
```diff
@@ -7,6 +7,45 @@ menu:
...
+- [Exec Input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/exec)(`exec`):
+  Fix truncated messages in debug mode; debug mode now shows full messages.
+- [IPMI Sensor Input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/ipmi_sensor)(`ipmi_sensor`):
+  Fix panic by implementing a length check to plugin.
+- [MySQL Input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/mysql)(`mysql`):
+  Fix the ability to handle ‘binary logs’ query for MySQL version 8.0+.
+- [NFS Client Input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/nfsclient)(`nfsclient`):
+  Fix integer overflow in fields received by mountstat.
+- [Ping Input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/ping)(`ping`):
+  Fix an issue when `method=”native”` by always calling `SetPrivileged(true)`.
```
Comment (kelseiv) [inline content/telegraf/v1.18/about_the_project/release-notes-changelog.md:28]: I'm not sure what makes sense to say here? Maybe:
Resolve error that prevented the agent from running when an unprivileged UDP ping was sent. Now, `SetPrivileged(true)` is always true in native mode to ensure a privileged ICMP ping is sent.

**Review by kelseiv — APPROVED** [review]

**Pull request merged**